.github/workflows/main.yml

```diff
@@ -9,12 +9,24 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 8, 11, 16 ]
```

Comment:
shall I test only Java 8?
Only 8 and 11?

Reply:
I'll reduce to Java 8 for now, since that would otherwise generate a LOT of builds